addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java

```diff
@@ -48,9 +48,9 @@
  * @version 1.1
  * @see https://www.veggiespam.com/ils/
  */
-public class ImageLocationScanner extends PluginPassiveScanner {
+public class ImageLocationScanRule extends PluginPassiveScanner {
 	private PassiveScanThread parent = null;
-	private static final Logger logger = Logger.getLogger(ImageLocationScanner.class);
+	private static final Logger logger = Logger.getLogger(ImageLocationScanRule.class);
 	private static final String MESSAGE_PREFIX = "imagelocationscanner.";
```

Comment:
No changes to the key?

Reply:
Well I didn't think I should rename the whole add-on.

I guess the keys don't have to be named after the add-on though.

I'll update those that are used by the scan rule.

Reply:
Fine as is, just checking.

Reply:
Okay